sycl/doc/extensions/experimental/sycl_ext_oneapi_device_architecture.asciidoc

```diff
@@ -128,25 +128,25 @@ enum class architecture : /* unspecified */ {
   intel_gpu_acm_g12,
   intel_gpu_pvc,
 
-  nvidia_gpu_sm20,
-  nvidia_gpu_sm30,
-  nvidia_gpu_sm32,
-  nvidia_gpu_sm35,
-  nvidia_gpu_sm37,
-  nvidia_gpu_sm50,
-  nvidia_gpu_sm52,
-  nvidia_gpu_sm53,
-  nvidia_gpu_sm60,
-  nvidia_gpu_sm61,
-  nvidia_gpu_sm62,
-  nvidia_gpu_sm70,
-  nvidia_gpu_sm72,
-  nvidia_gpu_sm75,
-  nvidia_gpu_sm80,
-  nvidia_gpu_sm86,
-  nvidia_gpu_sm87,
-  nvidia_gpu_sm89,
-  nvidia_gpu_sm90,
+  nvidia_gpu_sm_20,
+  nvidia_gpu_sm_30,
+  nvidia_gpu_sm_32,
+  nvidia_gpu_sm_35,
+  nvidia_gpu_sm_37,
+  nvidia_gpu_sm_50,
+  nvidia_gpu_sm_52,
+  nvidia_gpu_sm_53,
```

Comment:
Late in the game, but do we need entry for arch `sm_20` to `sm_37` ? IIRC `sm_50` or `sm_60` is the minimal version DPC++ can reliably support. Also `sm_50` is the minimal arch supported by the latest CUDA SDK (12).

Reply:
Thanks @Naghasan , removed am_20 to sm_37